data/successors/successor outfits.txt

```diff
@@ -538,60 +538,60 @@ outfit `"Veusa" Inductive Reverser`
 	description "	Because a reverse thruster must be facing forwards, it must be installed in the weapon section of your ship instead of the engine section."
 
 
-outfit "Multimodal Armor: Heavy"
+outfit "Multimodal Armor: Light"
 	category "Systems"
 	series "Special Systems"
 	index 11010
-	thumbnail "outfit/successor armor heavy"
+	thumbnail "outfit/successor armor light"
 	licenses
 		"High Houses"
-	"hull protection" .20
+	"hull protection" .1
+	"inertia reduction" .15
```

Comment:
Completely optional thought to tweak the Light Multimodal Armor slightly more towards its exploratory role to differentiate it more vs the .2 "hull protection" of Heavy Armor, but feel free to close this comment if you disagree. 
```suggestion
	"hull protection" .05
	"inertia reduction" .2
```

Reply:
I guess I'm just concerned about whether this stat line would be as valuable; 5% hull protection seems more useful than 5% inertia reduction, and I don't want the Light Armor to feel underpowered compared to the other options. I'll leave this open to see if others have any thoughts on the matter.

Reply:
I think that's fair, and doesn't look like there's much interest in a change here. I think this is a great PR and definitely don't want to hold it up with this comment -- feel free to close!